pkg/controller/importconfig/cluster_info.go

```diff
@@ -482,3 +486,62 @@ func getBootstrapSAName(clusterName string) string {
 	}
 	return bootstrapSAName
 }
+
+func filesToBytes(files ...string) []byte {
+	byteBuffer := new(bytes.Buffer)
+	for _, file := range files {
+		b, err := manifestFiles.ReadFile(file)
+		if err != nil {
+			// this should not happen, if happened, panic here
+			panic(err)
```

Comment:
why panic but not return err?

Reply:
This is copy from the previous way of implementation: https://github.com/stolostron/managedcluster-import-controller/blob/3e3a2f9c964e9b39a0be1584d1e4821fbc0a6fea/pkg/controller/importconfig/worker_default.go#L126

I guess previously we think the template format must be correct otherwise we should panic as soon as possible.

Fixed to return error now.